src/design-system/InputButton.tsx

```diff
@@ -1,24 +1,30 @@
 import React from "react";
 import styled from "styled-components";
 
-const StyledButton = styled.button`
+const StyledButton = styled.button<Props>`
   background: #00615c;
   font-size: 16px;
   border-radius: 12px;
   color: white;
   font-family: "Poppins", sans-serif;
   height: 48px;
-  width: 200px;
+  width: ${(props) =>
+    props.styles && props.styles.width ? props.styles.width : "200px"};
```

Comment:
I _think_ there is some neat TypeScript shorthand you can use here to bail out of `props.styles.width` if `styles` is undefined: `props.styles?.width` should evaluate to `undefined` in that case (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining)

Reply:
cool!